toshi_hazard_store/oq_import/oq_config.py

```diff
@@ -110,7 +110,10 @@ def process_hdf5(
             hdf5_file_name = calc_files[0]
             log.info(f"extracting {hdf5_file_name} from {hdf5_archive} into {subtask_folder}")
             myzip.extract(hdf5_file_name, subtask_folder)
-            hdf5_archive.unlink()  # delete the zip
+
+        # delete the archive
+        hdf5_archive.unlink()
```

Comment:
or is this the fix?

Reply:
Yes, the unlink (which is a file deletion) was happening inside the `with ....`context block. Which was clearly wrong, but wasn't causing issues on other platforms. I moved the the unlink outside of the context (where the hdf5_archive file is being used) and Windows is happy.